app/controllers/controllershelpers/HomeCardGenerator.scala

```diff
@@ -50,7 +51,8 @@ class HomeCardGenerator @Inject()(
       getTaxCalculationCard(taxCalculationStateCyMinusOne),
       getTaxCalculationCard(taxCalculationStateCyMinusTwo),
       getSelfAssessmentCard(saActionNeeded, currentTaxYear + 1),
-      getNationalInsuranceCard()
+      getNationalInsuranceCard(),
+      getAnnualTaxSummaryCard
```

Comment:
Has this got tests around it? (fine if not as its a bit of an odd function)

Reply:
Each of the functions called here are tested, and this just returns the result of these functions, so it should be fine